### PR TITLE
chore(samples): refactoring comments.

### PR DESCRIPTION
**Pull request by dfirova**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

**Review by chenlei1216 — APPROVED** [review]

File: samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java [hunk 1]
```diff
@@ -38,11 +38,11 @@ public class PurgeUserEvent {
 
   public static void main(String[] args)
       throws IOException, ExecutionException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
-    String projectId = ServiceOptions.getDefaultProjectId();
+    // TODO: Set projectId to your Google Cloud Platform project ID.
```
Comment (prash-mi) [inline samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java:41]: nit: do we want to replace TODO with TODO(developer)

File: samples/interactive-tutorials/src/main/java/events/RejoinUserEvent.java [hunk 2]
```diff
@@ -40,11 +40,11 @@ public class RejoinUserEvent {
...
-    String projectId = ServiceOptions.getDefaultProjectId();
+    // TODO: Set projectId to your Google Cloud Platform project ID.
+    String projectId = "my-project";
     String defaultCatalog =
         String.format("projects/%s/locations/global/catalogs/default_catalog", projectId);
-    // visitorId generated randomly.
+
     String visitorId = UUID.randomUUID().toString();
 
     callRejoinUserEvents(defaultCatalog, visitorId);
```
Comment (prash-mi) [inline samples/interactive-tutorials/src/main/java/events/RejoinUserEvent.java:50]: Do we want to rename it to `rejoinUserEvents`

File: samples/interactive-tutorials/src/main/java/events/WriteUserEvent.java [hunk 3]
```diff
@@ -38,11 +38,11 @@ public class WriteUserEvent {
...
       throws IOException, ExecutionException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
-    String projectId = ServiceOptions.getDefaultProjectId();
+    // TODO: Set projectId to your Google Cloud Platform project ID.
+    String projectId = "my-project";
     String defaultCatalog =
         String.format("projects/%s/locations/global/catalogs/default_catalog", projectId);
-    // visitorId generated randomly.
+
     String visitorId = UUID.randomUUID().toString();
```
Comment (prash-mi) [inline samples/interactive-tutorials/src/main/java/events/WriteUserEvent.java:46]: Do we want to add a line comment why are we using a random visitorId, or some guidance on what this value should be replaced with while writing user event?

File: samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java [hunk 4]
```diff
@@ -58,7 +58,7 @@ public static void callPurgeUserEvents(String defaultCatalog, String visitorId)
     try (UserEventServiceClient userEventServiceClient = UserEventServiceClient.create()) {
       PurgeUserEventsRequest purgeUserEventsRequest =
           PurgeUserEventsRequest.newBuilder()
-              // TO CHECK ERROR HANDLING SET INVALID FILTER HERE:
+              // To check error handling set invalid filter here:
               .setFilter(String.format("visitorId=\"%s\"", visitorId))
               .setParent(defaultCatalog)
               .setForce(true)
```
Comment (prash-mi) [inline samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java:64]: Do we want to add a comment mentioning why are we doing a  `setForce(true)`

**Comment by prash-mi** [comment]
@kolea2 - Do you want to take a look? I have left a few review comments

File: samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java (hunk 1 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java:41]: Replaced TODO with TODO(developer)

File: samples/interactive-tutorials/src/main/java/events/RejoinUserEvent.java (hunk 2 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/events/RejoinUserEvent.java:50]: In my opinion callRejoinUserEvents is descriptive of what the method is doing: calling the rejoinUserEvents method. So I don't think it is necessary for the name to change.

File: samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java (hunk 4 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java:64]: Added comment with the explanation.

**Comment by dfirova** [comment]
@prash-mi asking to review this request and also trigger tests.

**Comment by prash-mi** [comment]
@dfirova there appears to be code conflict, do you want to resolve it? I will run the tests again after that

**Comment by dfirova** [comment]
@yoshi-kokoro `error reading /root/.m2/repository/com/google/protobuf/protobuf-java/3.21.6/protobuf-java-3.21.6.jar; zip file is empty`

**Review by prash-mi — APPROVED** [review]

**Pull request merged**